content/posts/2023/pdf-root-files-preview.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Introducing PDF and ROOT file previews in REANA web"
+date: 2023-06-26T14:00:00+01:00
+---
+
+We are excited to announce that the REANA web interface now supports previewing of PDF and ROOT files.
```

Comment:
BTW, on the similar tune: it may be better to enlarge the scope of this blog post to present everything previewable, so to speak. Hence we could show an example of previewing HTML files by means of Snakemake report, say that it was there since quite some time, and only then speak about PDF and ROOT that were added recently. In this way, the blog post could serve as a comprehensive information for the file previewing topic, also for future reference. This will make its value apparent so that people wouldn't think "why another blog post for something that was already announced (albeit in a few lines)".  If we go for this, the blog post title should obviously change too, into something more generic, for example _"Previewing your workspace files"_.